src/rust/examples/slapper/src/requests/do_cool_stuff.rs

```diff
@@ -0,0 +1,52 @@
+use sessionless::hex::IntoHex;
+use sessionless::{PublicKey, Sessionless, Signature};
+use reqwest;
+use reqwest::blocking::Response;
+use colored::Colorize;
+use serde::{Deserialize};
+use crate::requests::server_config;
+use crate::requests::{WelcomeResponse, CoolnessResponse};
+
+pub fn do_cool_stuff(color: &String, SESSIONLESS: Sessionless, welcome_response: WelcomeResponse) {
```

Comment:
The argument `SESSIONLESS` should be renamed to `sessionless` as the Rust uses" **snake_case**" convention

Reply:
I'm cool with that, but I took this from the server example. Should we change it to lower case there too?

Reply:
> I took this from the server example

In the server example `SESSIONLESS` is a `static` item. Both `statics` and `consts` are supposed to be "UPPER_CASE". Regular variables, like the one here should use `snake_case`. I think it's designed this way to have a much better distinction between these items.